host-osx/Makefile

```diff
@@ -21,67 +21,98 @@ NATIVE_PATH = /Library/Google/Chrome/NativeMessagingHosts
 FIREFOX_PATH = /Library/Application\ Support/Mozilla/NativeMessagingHosts
 FFEXTENSION_PATH = /Library/Application\ Support/Mozilla/Extensions/\{ec8030f7-c20a-464f-9b0e-13a3a9e97384\}
 
+CHROME_EXTERNAL = $(TMP_CHROME)/Library/Application\ Support/Google/Chrome/
+CHROME_NATIVE = $(TMP_CHROME)/Library/Google/Chrome/
+FIREFOX_NATIVE = $(TMP_FIREFOX)/Library/Google/Chrome/
+FIREFOX_NATIVEHOSTS = $(TMP_FIREFOX)/Library/Application\ Support/Mozilla/
+FIREFOX_EXTENSIONS = $(TMP_FIREFOX)/Library/Application\ Support/Mozilla/Extensions
+
 BUILD_NUMBER ?= 0
 include ../VERSION.mk
 
 SIGNER ?= Riigi Infosüsteemi Amet
 TMPROOT = $(PWD)/tmp
+TMP_CHROME = $(PWD)/tmp_chrome
+TMP_FIREFOX = $(PWD)/tmp_firefox
 POLICYROOT = $(PWD)/tmp2
 POLICYSIGNED = $(PWD)/tmpsigned
 TARGET = $(TMPROOT)/Library/Google/Chrome/NativeMessagingHosts/chrome-token-signing.app
-PKG = chrome-token-signing.pkg
-POLICY = chrome-token-signing-policy.pkg
+PKG_CHROME = token-signing-chrome.pkg
+PKG_FIREFOX = token-signing-firefox.pkg
+POLICY = token-signing-policy.pkg
 
 PROJ = xcodebuild -project chrome-token-signing.xcodeproj -configuration Release VERSION=$(VERSION) BUILD_NUMBER=$(BUILD_NUMBER)
 
 $(TARGET): *.mm
 	$(PROJ) install DSTROOT=$(TMPROOT)
+	$(MAKE) separate
 
-codesign: $(TARGET)
+codesign: $(TARGET) 
 	codesign -f --entitlements chrome-token-signing.entitlements --options=runtime -s "Developer ID Application: $(SIGNER)" $<
 
 clean:
 	$(PROJ) clean
 	git clean -dfx
 
-$(PKG): $(TARGET)
+$(PKG_CHROME): $(TARGET)
+	pkgbuild --version $(VERSION) \
+		--root $(TMP_CHROME) \
+		--identifier ee.ria.token-signing-chrome \
+		--install-location / \
+		$@
+
+$(PKG_FIREFOX): $(TARGET)
 	pkgbuild --version $(VERSION) \
-		--root $(TMPROOT) \
-		--identifier ee.ria.chrome-token-signing \
+		--root $(TMP_FIREFOX) \
+		--identifier ee.ria.token-signing-firefox \
 		--install-location / \
 		$@
 
 $(POLICY):
 	mkdir -p $(POLICYROOT)
 	pkgbuild --version $(VERSION) \
 		--root $(POLICYROOT) \
-		--identifier ee.ria.chrome-token-signing-policy \
+		--identifier ee.ria.token-signing-chrome-policy \
 		--install-location / \
 		--scripts scripts \
 		$@
 
-pkg: $(PKG) $(POLICY)
+pkg: $(PKG_CHROME) $(PKG_FIREFOX) $(POLICY)
 
 signed: codesign
 	pkgbuild --version $(VERSION) \
-		--root $(TMPROOT) \
-		--identifier ee.ria.chrome-token-signing \
+		--root $(TMP_CHROME) \
+		--identifier ee.ria.token-signing-chrome \
 		--install-location / \
 		--sign "Developer ID Installer: $(SIGNER)" \
-		$(PKG)
+		$(PKG_CHROME)
+	pkgbuild --version $(VERSION) \
```

Comment:
Can you use variable to remove some duplicate parameters.
See PROJ = xcodebuild